ec/ecdatasource/deploymentsdatasource/expanders.go

```diff
@@ -45,21 +44,19 @@ func expandFilters(d *schema.ResourceData) (*models.SearchRequest, error) {
 	if depTemplateID != "" {
 		esPath := "resources.elasticsearch"
 		tplTermPath := esPath + ".info.plan_info.current.plan.deployment_template.id"
-		tplID := ec.String(depTemplateID)
 
-		queries = append(queries, newNestedTermQuery(esPath, tplTermPath, tplID))
+		queries = append(queries, newNestedTermQuery(esPath, tplTermPath, depTemplateID))
 	}
 
 	healthy := d.Get("healthy").(string)
 	if healthy != "" {
-		h, err := strconv.ParseBool(healthy)
-		if err != nil {
-			return nil, err
+		if healthy != "true" && healthy != "false" {
```

Comment:
This used to be covered by a parse into bool which isn't necessary anymore. But since this is covered by a unit test I decided to just add a manual check instead.